src/pages/lib-v1/project-setup/create-application/q/platform/[platform].mdx

```diff
@@ -29,37 +29,247 @@ import flutter2 from '/src/fragments/lib-v1/project-setup/native_common/create-a
 
 <InlineFilter filters={['js']}>
 
-import js0 from '/src/fragments/lib-v1/start/getting-started/angular/setup.mdx';
```

Comment:
are these fragments used elsewhere? If not, then can you also commit the file deletions? #defrag